### PR TITLE
define and apply YAPF formatting 

### PR DESCRIPTION
**Pull request by Borda**
### Description
adding YAPF formatting

### Status
**Ready/Work in progress/Hold**

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or new feature that would cause existing functionality to change)
- [x] New tests added to cover the changes
- [ ] Docstrings/Documentation updated

### PR Implementer
This is a small checklist for the implementation details of this PR.

If there are any questions regarding code style or other conventions check out our 
[summary](https://github.com/kornia/kornia/blob/master/CONTRIBUTING.rst).

- [x] Did you discuss the functionality or any breaking changes before ?
- [x] **Pass all tests**: did you test in local ? `make test`
- [x] Unittests: did you add tests for your new functionality ?
- [x] Documentations: did you build documentation ? `make build-docs`
- [x] Implementation: is your code well commented and follow conventions ? `make lint`
- [x] Docstrings & Typing: has your code documentation and typing ? `make mypy`
- [x] Update notebooks & documentation if necessary

### KorniaTeam
<details>
  <summary>KorniaTeam workflow</summary>
  
  - [ ] Assign correct label
  - [ ] Assign PR to a reviewer
  - [ ] Does this PR close an Issue? (add `closes #IssueNumber` at the bottom if 
        not already in description)

</details>

### Reviewer
<details>
  <summary>Reviewer workflow</summary>

  - [ ] Do all tests pass? (Unittests, Typing, Linting, Documentation, Environment)
  - [ ] Does the implementation follow `kornia` design conventions?
  - [ ] Is the documentation complete enough ?
  - [ ] Are the tests covering simple and corner cases ?
 
</details>

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -158,12 +158,14 @@ This section provides general guidance for developing code for the project. The
 - Format your code:
 
   - We follow `PEP8 style guide <https://www.python.org/dev/peps/pep-0008>`_.
-  - Use ``autopep`` to autoformat: https://pypi.org/project/autopep8/#id3 .
+  - Use ``yapf`` to autoformat: https://github.com/google/yapf
     For doing so, just run the following from the repository root
 
   .. code:: bash
 
     make autopep8
```
Comment (edgarriba) [inline CONTRIBUTING.rst:166]: this goes out I believe

**Comment by Borda** [comment]
I have split the apply YAFL per 3 commits per root folders 608b079 2aa353c 8ce2c5f

File: kornia/augmentation/__init__.py [hunk 2]
```diff
@@ -1,7 +1,4 @@
-from .base import (
-    AugmentationBase2D,
-    AugmentationBase3D
-)
+from .base import (AugmentationBase2D, AugmentationBase3D)
```
Comment (Borda) [inline kornia/augmentation/__init__.py:1]: @edgarriba this can be addressed with Isort #1040

**Review by edgarriba — APPROVED** [review]
LGTM. Some crashing tests for formatting though

**Comment by Borda** [comment]
> LGTM. Some crashing tests for formatting though

Yep, going to check it out...

**Comment by Borda** [comment]
@edgarriba seems to e ready to land...

**Review by edgarriba — APPROVED** [review]

**Pull request merged**